distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -51,6 +51,9 @@ type Point_Data
     ## PRIVATE
     Row_Number
 
+    ## PRIVATE
+    Colour
```

Comment:
I feel like having both constructors `Color` and `Colour` is quite confusing. Which one should I use? Are they equivalent? Looking at the code below:
```
        Point_Data.Color ->
            colour_col = Point_Data.Colour.lookup_in table
```
before I noticed there's 2 different constructors I was pretty sure this was a typo (once `Color` and once `Colour`). It's so so easy to confuse the two.

Wouldn't it be much cleaner to keep a single constructor but modify the lookup/fallback mechanism to accept columns with both names? E.g. we can keep only the `Color` constructor but allow it to detect columns named `color` as well as `colour`. I would very much prefer such solution if possible.

Reply:
Yeah I agree it isn't clear, I'll look into how we can see if it will detect both!